package.json

```diff
@@ -75,6 +75,7 @@
   "scripts": {
     "build": "gulp build",
     "lint": "gulp lint sass-lint",
+    "package": "cd addon && npm run package",
```

Comment:
Or, FWIW, you might even leave it out of the root `package.json` altogether... it's a little redundant with the same command in `addon/package.json`